pennylane/workflow/__init__.py

```diff
@@ -29,19 +29,6 @@
     ~workflow.get_transform_program
     ~workflow.get_best_diff_method
 
-Supported interfaces
-~~~~~~~~~~~~~~~~~~~~
-
-.. autosummary::
-    :toctree: api
-
-    ~workflow.interfaces.autograd
-    ~workflow.interfaces.jax
-    ~workflow.interfaces.jax_jit
-    ~workflow.interfaces.tensorflow
-    ~workflow.interfaces.tensorflow_autograph
-    ~workflow.interfaces.torch
-
```

Comment:
Why was this removed?

Reply:
@albi3ro

Reply:
We were rendering the documentation for a torch object that has deprecation warnings in it's docstrings. Or something.

Basically, these things are really internal and don't really need to render in the docs, and trying to render them caused issues.